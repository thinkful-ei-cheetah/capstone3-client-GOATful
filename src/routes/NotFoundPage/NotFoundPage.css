.not-found-page {
  min-height: 400px;
  display: flex;
  justify-content: center;
  align-items: center;
  text-align: center;
}