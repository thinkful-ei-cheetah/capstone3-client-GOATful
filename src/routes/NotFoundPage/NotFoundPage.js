import React, { Component } from 'react'
import './NotFoundPage.css'

export default class NotFound extends Component {
  render() {

    return (
      <section className="not-found-page">
        <div>
          <h2>404</h2>
          <p>The page you requested no longer exists!</p>
        </div>
      </section>
    );
  }
}
