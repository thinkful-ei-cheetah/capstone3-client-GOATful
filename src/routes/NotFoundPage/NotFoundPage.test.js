import React from 'react';
import ReactDOM from 'react-dom';
import NotFoundPage from './NotFoundPage';

describe('NotFoundPage Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(< NotFoundPage />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
})
