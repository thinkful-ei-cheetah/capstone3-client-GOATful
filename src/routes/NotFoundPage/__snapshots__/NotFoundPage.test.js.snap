// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage Component renders the UI as expected 1`] = `
<section
  className="not-found-page"
>
  <div>
    <h2>
      404
    </h2>
    <p>
      The page you requested no longer exists!
    </p>
  </div>
</section>
`;
